Tests/MapboxMapsTests/Location/LocationManagerTests.swift

```diff
@@ -37,4 +37,27 @@ internal class LocationManagerTests: XCTestCase {
         XCTAssertNil(locationManager.delegate)
         XCTAssertFalse(locationManager.showUserLocation)
     }
+
+    func testLocationManagerPuckUpdate() {
+        var locationOptions = LocationOptions()
+        locationOptions.showUserLocation = true
+        let locationManager = LocationManager(locationOptions: locationOptions,
+                                              locationSupportableMapView: locationSupportableMapMock)
+
+        var locationOptions2 = LocationOptions()
+        locationOptions2.puckType = .puck2D(Puck2DConfiguration(scale: .constant(2.0)))
+        locationOptions2.showUserLocation = true
+        locationManager.updateLocationOptions(with: locationOptions2)
+        XCTAssertEqual(locationManager.locationPuckManager?.puckType, locationOptions2.puckType)
+
+        var locationOptions3 = LocationOptions()
+        locationManager.updateLocationOptions(with: locationOptions3)
+        XCTAssertNil(locationManager.locationPuckManager)
```

Comment:
could we add `locationOptions3.showUserLocation = false` to this section so that this test doesn't depend on the default value for `showUserLocation`?

Reply:
Thanks, updated